src/domain/space/space/space.service.license.ts

```
@@ -169,12 +169,24 @@ export class SpaceLicenseService {
           }
           break;
         case LicenseEntitlementType.SPACE_FLAG_WHITEBOARD_MULTI_USER:
-          const createInnovationHub =
+          const createWhiteboard =
             await this.licenseEngineService.isEntitlementGranted(
               LicenseEntitlementType.SPACE_FLAG_WHITEBOARD_MULTI_USER,
               levelZeroSpaceAgent
             );
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)